docs/DeveloperGuide.md

```diff
@@ -790,3 +790,20 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Effort**
+
+* As we were building off of AB3, we were able to reuse the model class and some of the commands.
```

Comment:
Small issues:
1. "some of the model classes" instead of model class bc there is no single model class only a model package. Or you can say model package
2. We reused all the commands but just modified them to suit our needs right? Maybe you can phrase it more like " We were able reuse the model class.
3. Can include reuse of extensive test cases and utility in the testutil package, since that did save a lot of work

Reply:
thankss